docs/appendix/apis/_index.md

```diff
@@ -15,387 +15,72 @@ no_list: true
 ---
 
 Every Viam {{< glossary_tooltip term_id="resource" text="resource" >}} exposes an [Application Programming Interface (API)](https://en.wikipedia.org/wiki/API) described through [protocol buffers](https://developers.google.com/protocol-buffers).
-You can think of this as a description of how you can interact with that resource.
-Different models of resources implement the same API, which [Viam SDKs expose](/architecture/machine-to-machine-comms/), allowing you to control different models of resource types with a consistent interface.
 
 The API methods provided by the SDKs for each of these resource APIs wrap gRPC client requests to the machine when you execute your program, providing you a convenient interface for accessing information about and controlling the {{< glossary_tooltip term_id="resource" text="resources" >}} you have [configured](/configure/) on your machine.
 
-## Robot management APIs
+## Machine and fleet APIs
 
-### Robot API
+{{< cards >}}
+{{% manualcard link="/appendix/apis/fleet/" title="Fleet Management API" %}}
 
-All machines support the following methods through the [robot API](/appendix/apis/robot/):
+Create and manage organizations, locations, and individual machines, fragments and permissions.
```

Comment:
[opt] It is a bit unclear why I can do things with individual machines both through the fleet mgmnt api as well as with the machine api. Maybe make it more clear that fleet mgmnt has to do with the app client vs machine mgmnt is more local or low-level (if that's the case)?

Reply:
```suggestion
Create and manage organizations, locations, and machines, get logs from individual machines, and manage fragments and permissions.
```